src/Ng/Arguments.cs

```diff
@@ -133,6 +133,11 @@ public static class Arguments
         /// Default value of <see cref="PackageStatusFolder"/>.
         /// </summary>
         public const string PackageStatusFolderDefault = "status";
+
+        /// <summary>
+        /// If the queue contains more messages than this, the job will not requeue any invalid packages.
+        /// </summary>
+        public const string MaxRequeueQueueSize = "maxQueueSize";
```

Comment:
`MaxQueueSize`, to be consistent with the parameter name?

Reply:
The pattern in this file is that the arguments start with lowercase. For example, `StorageAccountName` maps to `storageAccountName`.